### PR TITLE
Experiment around block identification

### PR DESCRIPTION
**Pull request by by-dam**
As a step belongs to many windows, averages the classification results of all the windows containing that step.
Then aggregates together all consecutive steps with the same outcome.

### CODE REVIEW

**Comment by duchoang** [comment]
:+1:

File: MuvrKit/MKClassifier.swift [hunk 1]
```diff
@@ -93,40 +106,116 @@ public struct MKClassifier {
...
         }
-        
+        return cews
+    }
+    
+    ///
+    /// Compute the probability for every window step by averaging the probabilities over all windows containing the step
+    ///
+    func classifySteps(windows: [MKClassifiedExerciseWindow], samplesPerSecond: UInt) -> [MKClassifiedExerciseBlock] {
+        let duration = Double(windowStepSize) / Double(samplesPerSecond)
```
Comment (tmbo) [inline MuvrKit/MKClassifier.swift]: Won't the duration be wrong for the last window? Since the last window will be 8 seconds long?

Comment (by-dam) [inline MuvrKit/MKClassifier.swift]: Even the last window is split into steps so the duration is always 0.2s

Comment (tmbo) [inline MuvrKit/MKClassifier.swift]: Alright :)

**Pull request merged**